app/robots/iCubGenova09/leftFingersHapticRetargetingParams.ini

```diff
@@ -127,7 +152,7 @@ axesJointsCoupled     1
 
 # in case a calibration phase necessary before starting teleoperating the robot,
 # otherwise read the calibration matrix from the config file
-doCalibration           1
+doCalibration           0
```

Comment:
@RiccardoGrieco @lrapetti do you remember why this modification was necessary?

Reply:
As far as I remember this should be a last-minute modification when we tried to remove the haptic force feedback. I would probably discard this change. (this code is not the one that is currently used in the robot right?)